### PR TITLE
Suave functional

### PR DESCRIPTION
**Pull request by varasev**
## Motivation

This PR adds the following Suave-specific features:
- extra fields on the transaction page (`Execution Node`, `Allowed Peekers`, and wrapped ethereum transaction)
- a list of transactions filtered by execution node (`/api/v2/transactions/execution-node/:execution_node_hash_param` endpoint)
- `SUAVE_BID_CONTRACTS` env variable to define comma-separated list of Bid Contracts (to retrieve `BidEvent` from logs which is used to get `Allowed Peekers`)
- `CHAIN_TYPE=suave` env variable

Parsing of `confidentialComputeResult` field of `eth_getTransactionByHash` wasn't added as its value always matches the `input` which is already parsed by standard Blockscout functionality.

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [ ] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [x] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex [hunk 1]
```diff
@@ -153,6 +169,71 @@ defmodule EthereumJSONRPC.Transaction do
   """
   @spec elixir_to_params(elixir) :: params
 
+  def elixir_to_params(
```
Comment (vbaranov) [inline apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex:172]: Let's indicate in the doc, that this is Suave -specific

File: apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex [hunk 2]
```diff
@@ -252,6 +333,67 @@ defmodule EthereumJSONRPC.Transaction do
     end
   end
 
+  def elixir_to_params(
```
Comment (vbaranov) [inline apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex:336]: Let's indicate in the doc, that this is Suave -specific

**Review by vbaranov — CHANGES_REQUESTED** [review]
The branch is not backward-compatible with the running non-suave chain. For instance, this error appear (and probably more):
```
2023-09-28T19:37:59.253 application=indexer fetcher=block_catchup first_block_number=5956031 last_block_number=5956031 step=fork_transactions [error] failed to insert: %{exception: %Postgrex.Error{message: nil, postgres: %{code: :undefined_column, file: "parse_relation.c", line: "3408", message: "column st0.execution_node_hash does not exist", pg_code: "42703", position: "1354", routine: "errorMissingColumn", severity: "ERROR", unknown: "ERROR"}, connection_id: 4947, query: "UPDATE \"transactions\" AS t0 SET \"block_hash\" = NULL, \"block_number\" = NULL, \"gas_used\" = NULL, \"cumulative_gas_used\" = NULL, \"index\" = NULL, \"status\" = NULL, \"error\" = NULL, \"max_priority_fee_per_gas\" = NULL, \"max_fee_per_gas\" = NULL, \"type\" = NULL, \"updated_at\" = $1 FROM (SELECT st0.\"hash\" AS \"hash\", st0.\"block_number\" AS \"block_number\", st0.\"cumulative_gas_used\" AS \"cumulative_gas_used\", st0.\"earliest_processing_start\" AS \"earliest_processing_start\", st0.\"error\" AS \"error\", st0.\"gas\" AS \"gas\", st0.\"gas_price\" AS \"gas_price\", st0.\"gas_used\" AS \"gas_used\", st0.\"index\" AS \"index\", st0.\"created_contract_code_indexed_at\" AS \"created_contract_code_indexed_at\", st0.\"input\" AS \"input\", st0.\"nonce\" AS \"nonce\", st0.\"r\" AS \"r\", st0.\"s\" AS \"s\", st0.\"status\" AS \"status\", st0.\"v\" AS \"v\", st0.\"value\" AS \"value\", st0.\"revert_reason\" AS \"revert_reason\", st0.\"max_priority_fee_per_gas\" AS \"max_priority_fee_per_gas\", st0.\"max_fee_per_gas\" AS \"max_fee_per_gas\", st0.\"type\" AS \"type\", st0.\"has_error_in_internal_txs\" AS \"has_error_in_internal_txs\", st0.\"old_block_hash\" AS \"old_block_hash\", st0.\"inserted_at\" AS \"inserted_at\", st0.\"updated_at\" AS \"updated_at\", st0.\"block_hash\" AS \"block_hash\", st0.\"from_address_hash\" AS \"from_address_hash\", st0.\"to_address_hash\" AS \"to_address_hash\", st0.\"created_contract_address_hash\" AS \"created_contract_address_hash\", st0.\"execution_node_hash\" AS \"execution_node_hash\", st0.\"wrapped_type\" AS \"wrapped_type\", st0.\"wrapped_nonce\" AS \"wrapped_nonce\", st0.\"wrapped_gas\" AS \"wrapped_gas\", st0.\"wrapped_gas_price\" AS \"wrapped_gas_price\", st0.\"wrapped_max_priority_fee_per_gas\" AS \"wrapped_max_priority_fee_per_gas\", st0.\"wrapped_max_fee_per_gas\" AS \"wrapped_max_fee_per_gas\", st0.\"wrapped_value\" AS \"wrapped_value\", st0.\"wrapped_input\" AS \"wrapped_input\", st0.\"wrapped_v\" AS \"wrapped_v\", st0.\"wrapped_r\" AS \"wrapped_r\", st0.\"wrapped_s\" AS \"wrapped_s\", st0.\"wrapped_hash\" AS \"wrapped_hash\", st0.\"wrapped_to_address_hash\" AS \"wrapped_to_address_hash\" FROM \"transactions\" AS st0 WHERE ((FALSE)) OR ((st0.\"block_hash\" != $2) AND (st0.\"block_number\" = $3)) ORDER BY st0.\"hash\" FOR NO KEY UPDATE) AS s1 WHERE (t0.\"hash\" = s1.\"hash\") RETURNING s1.\"hash\", s1.\"block_number\", s1.\"cumulative_gas_used\", s1.\"earliest_processing_start\", s1.\"error\", s1.\"gas\", s1.\"gas_price\", s1.\"gas_used\", s1.\"index\", s1.\"created_contract_code_indexed_at\", s1.\"input\", s1.\"nonce\", s1.\"r\", s1.\"s\", s1.\"status\", s1.\"v\", s1.\"value\", s1.\"revert_reason\", s1.\"max_priority_fee_per_gas\", s1.\"max_fee_per_gas\", s1.\"type\", s1.\"has_error_in_internal_txs\", s1.\"old_block_hash\", s1.\"inserted_at\", s1.\"updated_at\", s1.\"block_hash\", s1.\"from_address_hash\", s1.\"to_address_hash\", s1.\"created_contract_address_hash\", s1.\"execution_node_hash\", s1.\"wrapped_type\", s1.\"wrapped_nonce\", s1.\"wrapped_gas\", s1.\"wrapped_gas_price\", s1.\"wrapped_max_priority_fee_per_gas\", s1.\"wrapped_max_fee_per_gas\", s1.\"wrapped_value\", s1.\"wrapped_input\", s1.\"wrapped_v\", s1.\"wrapped_r\", s1.\"wrapped_s\", s1.\"wrapped_hash\", s1.\"wrapped_to_address_hash\""}}. Retrying.
```

File: apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex (hunk 1 above)
Comment (varasev) [reply apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex:173]: Done in https://github.com/blockscout/blockscout/pull/8556/commits/3bf6579c238bc09ee564f07135e17859478a1225

File: apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex (hunk 2 above)
Comment (varasev) [reply apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex:347]: Done in https://github.com/blockscout/blockscout/pull/8556/commits/3bf6579c238bc09ee564f07135e17859478a1225

**Review by vbaranov — COMMENTED** [review]
@varasev please rebase onto master branch

**Comment by varasev** [comment]
> @varasev please rebase onto master branch

Done

**Review by vbaranov — COMMENTED** [review]
Please rebase onto master branch one more time 🙏 . We're are close to merge it.

File: apps/explorer/config/test.exs [hunk 3]
```diff
@@ -49,6 +49,15 @@ for repo <- [Explorer.Repo.PolygonEdge, Explorer.Repo.RSK] do
     queue_target: 1000
 end
 
+config :explorer, Explorer.Repo.Suave,
```
[the remark this answers is not in the document]
Comment (varasev) [reply apps/explorer/config/test.exs]: Done in https://github.com/blockscout/blockscout/commit/fc1f05f8ccb0492a7fbf91816b3ffc5dc5e319ca

**Review by vbaranov — APPROVED** [review]

**Pull request merged**